iot/mqtt_example/system-test/cloudiot_mqtt_example.test.js

```diff
@@ -146,3 +146,23 @@ test(`should send state message`, async t => {
   );
   await tools.runAsync(`${helper} deleteRegistry ${localRegName}`, cwd);
 });
+
+test(`should receive command message`, async t => {
+  const localDevice = `test-rsa-device`;
+  const localRegName = `${registryName}-rsa256`;
```

Comment:
/FYI I have seen the tests run in parallel which could cause flakiness if the names are not distinct per registry+device.